src/auto/injector.js

```diff
@@ -819,7 +818,8 @@ function createInjector(modulesToLoad, strictDi) {
         try {
           path.unshift(serviceName);
           cache[serviceName] = INSTANTIATING;
-          return cache[serviceName] = factory(serviceName, caller);
+          cache[serviceName] = factory(serviceName, caller);
```

Comment:
I actually prefer this way most of the time, the intention is clearer and the JIT should optimize it anyway. I changed most of the things like I did here.

I think the parens "hack" is nice in case of regex matching where it's not so easy to change without introducing a repeated line of code.